source/lambda/apiprocessor/redaction.py

```diff
@@ -33,7 +32,7 @@
 MAXIMUM_NUM_OF_HEADERS_FOOTERS = 3
 
 # we expect a redacted item to have 6 keys: page, top, left, width, height label
```

Comment:
Can we change the comment to say "we expect redacted item to have 5 keys and label is optional"

Reply:
done